jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/XGBoost.java

```diff
@@ -76,6 +76,19 @@ public static Booster train(
           IObjective obj,
           IEvaluation eval,
           int earlyStoppingRound) throws XGBoostError {
+    return train(dtrain, params, round, watches, metrics, obj, eval, earlyStoppingRound, null);
+  }
+
+  public static Booster train(
```

Comment:
would you help to add comments in these train() methods?

Reply:
Done.